### DISCUSSION TITLE
Wiki links are converted to markdown links and encoded even though the setting is off

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#141] [Q&A — Plugin] [answered]
I have a [repo](https://github.com/ooker777/mkdocs-hyperlink-bug) in which all file and folder paths have Unicode characters. Say in file A we have a link to file B. Here are how the link of B in A is converted:

- In my vault: `[[:gem: Giới thiệu về Obsidian]]`
- [In the `docs` folder](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/docs/index.md): `[:gem: Giới thiệu về Obsidian]:(./%F0%9F%93%9C%20T%C3%A0i%20nguy%C3%AAn/%F0%9F%92%8E%20Gi%E1%BB%9Bi%20thi%E1%BB%87u%20v%E1%BB%81%20Obsidian/index.md)` 
- [In the `site` folder](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/site/index.html): `<a href="%25F0%259F%2593%259C%2520T%25C3%25A0i%2520nguy%25C3%25AAn/%25F0%259F%2592%258E%2520Gi%25E1%25BB%259Bi%2520thi%25E1%25BB%2587u%2520v%25E1%25BB%2581%2520Obsidian/index.md">💎 Giới thiệu về Obsidian</a>` 

As the result the links are broken. Checking the URL encoder I see that Mkdocs just encodes the already encoded link.

For reference, here is the B file in [docs folder](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/docs/%F0%9F%93%9C%20T%C3%A0i%20nguy%C3%AAn/%F0%9F%92%8E%20Gi%E1%BB%9Bi%20thi%E1%BB%87u%20v%E1%BB%81%20Obsidian/index.md), [site folder](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/site/📜%20Tài%20nguyên/💎%20Giới%20thiệu%20về%20Obsidian/index.html) 

The plugin setting:
```js
{
  "github": {
    "branch": "main",
    "automaticallyMergePR": true,
    "api": {
      "tiersForApi": "Github Free/Pro/Team (default)",
      "hostname": ""
    },
    "worflow": {
      "customCommitMsg": "[PUBLISHER] Merge",
      "workflowName": ""
    }
  },
  "upload": {
    "behavior": "obsidian",
    "defaultName": "docs",
    "rootFolder": "",
    "yamlFolderKey": "",
    "frontmatterTitle": {
      "enable": false,
      "key": "title"
    },
    "replaceTitle": [],
    "replacePath": [],
    "autoclean": {
      "enable": true,
      "excluded": [
        "asset"
      ]
    },
    "folderNote": {
      "enable": true,
      "rename": "index.md"
    },
    "metadataExtractorPath": ""
  },
  "conversion": {
    "hardbreak": true,
    "dataview": true,
    "censorText": [],
    "tags": {
      "inline": true,
      "exclude": [],
      "fields": []
    },
    "links": {
      "internal": false,
      "unshared": false,
      "wiki": false,
      "slugify": false
    }
  },
  "embed": {
    "attachments": true,
    "keySendFile": [],
    "notes": false,
    "folder": ""
  }
}
```

**Comment by Mara-Li** [comment]
So the bug appear after the conversion by the plugin?

**Reply by ooker777** [reply]
I suppose. Which part in the plugin enables the encoding?

**Reply by Mara-Li** [reply]
I saw that you disable markdown links conversion. Can I get the original file after the conversion?

**Reply by ooker777** [reply]
you mean [the one in the `docs` folder](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/docs/index.md)?

**Reply by Mara-Li** [reply, marked as answer]
Are you sure you disabled the conversion from wikilinks ?

**Reply by ooker777** [reply]
hmm it seems that either the Digital Garden plugin or the WordPress plugin conflicts with this. Removing them makes this work, although I haven't double-checked if this is the source of the bug. The refresh issue seems to also work again

**Reply by ooker777** [reply]
nevertheless, should the plugin not to encode the link with using the converting to markdown link option? Because Mkdocs has already encode it.
